tools/internal_ci/linux/grpc_e2e_performance_v2.sh

```diff
@@ -19,5 +19,35 @@ cd $(dirname $0)/../../..
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 
-echo "TODO: Add gRPC OSS Benchmarks here..."
+# This is to ensure we can push and pull images from gcr.io. We do not
+# necessarily need it to run load tests, but will need it when we employ
+# pre-built images in the optimization.
+gcloud auth configure-docker
+
+# Connect to benchmarks-prod cluster.
+gcloud config set project grpc-testing
+gcloud container clusters get-credentials benchmarks-prod \
+    --zone us-central1-b --project grpc-testing
+
+# This is subject to change. Runs a single test and does not wait for the
+# result.
+tools/run_tests/performance/loadtest_config.py -l go \
+    -t ./tools/run_tests/performance/templates/loadtest_template_basic_all_languages.yaml \
+    -s client_pool=workers-8core -s server_pool=workers-8core \
+    -s big_query_table=grpc-testing.e2e_benchmarks.experimental_results \
+    -s timeout_seconds=900 --prefix="kokoro-test" -u "$(date +%Y%m%d%H%M%S)" \
+    -r go_generic_sync_streaming_ping_pong_secure -o ./loadtest.yaml
+
```

Comment:
A lot of the fields so far are hardcoded here. Does that mean in continuous build, we will always have a fix set of test to run? If someone wanted to run a different set of the tests continuously, they would need to start a new Kokoro job.

Reply:
Yes or edit their own copy of it before launching it. The one running continuously is always expected to be the same unless you update this script.